app/scripts/directives/unit.coffee

```diff
@@ -22,11 +22,8 @@ angular.module('swarmApp').directive 'unit', ($log, game, commands, options, uti
       #util.utcdoy 1000 * secs
       if isFinite estimate.val
         nonlinear = if not (estimate.unit?.isVelocityConstant?() ? true) then 'less than ' else ''
```

Comment:
I commented before about prefixing durations with 'less than' if the estimate's not exact (involves units-producing-units). That's still important; I don't want people reporting incorrect times as bugs. (Hopefully I'll improve the estimates soon so it's not necessary.)